.builders/scripts/build_dependencies.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+
+set -exu
+
+build_dependencies_file="/home/build_dependencies.txt"
+constraints_file="/home/constraints.txt"
+wheels_dir="/home/wheels_py3"
+patches_folder="/home/patches"
+
+touch "${constraints_file}"
+echo "bcrypt < 4.1.0" >> "${constraints_file}"
```

Comment:
What's this for?

Reply:
https://github.com/DataDog/datadog-agent/pull/21147

Reply:
Okay thanks, let's add a todo comment for removal